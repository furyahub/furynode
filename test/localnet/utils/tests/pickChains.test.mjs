import { pickChains } from "../pickChains.mjs";

test("pick chains", () => {
  const result = pickChains({ chain: "furynode,cosmos,akash" });

  expect(result).toMatchSnapshot();
});
