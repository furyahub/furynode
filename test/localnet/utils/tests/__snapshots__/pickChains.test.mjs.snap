// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pick chains 1`] = `
Array [
  "furynode",
  "cosmos",
  "akash",
]
`;
